themes/index.js

```diff
@@ -161,6 +161,12 @@ const themes = {
     text_color: "a6accd",
     bg_color: "292d3e",
   },
+  "match-with-readme": {
```

Comment:
the name match-with-readme is too specific, maybe try "grayscale", or just "black-white",  or "simple-dark"